sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -283,6 +292,49 @@ public void close()
     planner.close();
   }
 
+  private SqlNode processStatementList(SqlNode root, String sql)
```

Comment:
This method could use some javadoc about what is meant by "process" -- what is it looking for and what does it return?

Reply:
added javadocs